flows/definition/legacy.go

```diff
@@ -296,34 +308,33 @@ func createAction(baseLanguage utils.Language, a legacyAction, fieldMap map[stri
 			fieldMap[a.Value] = fieldUUID
 		}
 
-		translated, _ := excellent.TranslateTemplate(a.Value)
+		migratedValue, _ := excellent.MigrateTemplate(a.Value)
+
+		// flows now have different action for name changing
+		if a.Field == "name" || a.Field == "first_name" {
+			// we can emulate setting only the first name with an expression
+			if a.Field == "first_name" {
+				migratedValue = fmt.Sprintf("%s @(word_slice(contact.name, 2, -1))", migratedValue)
```

Comment:
https://github.com/nyaruka/rapidpro/blob/1ba705623953e28d6378957b55b41f4affc2ecf1/temba/flows/models.py#L6101-L6101

I'm planning to build a time machine and go back a few years and warn us not to add things like this